samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs

```diff
@@ -102,7 +102,9 @@ static void Main(string[] args)
             //</snippet11>
 
             //<snippet12>
-            foreach (var ch in laugh)
+            // Do not use implicit typing to determine the type of the loop
+            // variable in foreach loops.
```

Comment:
```suggestion
```
Please remove the comments.  I'm fine with the explicit typing by, no need to comment it.

Reply:
Some additional context on the comments: We limit comments in our samples because comment text is not localized. For that reason we prefer explanatory text in the accompanying article.

Reply:
Aight, was just emulating the samples above in the code file. :) The explanatory text is already in the doc.